Competition2017/tests/competition/subsystems/climb/ClimbingSubsystemTest.java

```diff
@@ -8,6 +8,13 @@
     @Test
     public void stopTest(){
         
+        climb.ascend();
+        
```

Comment:
You should add `verifyClimbingSetpoints()` here to check that it is actually ascending, just in case =]